platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java

```diff
@@ -500,4 +511,619 @@ public void test13_ExportPageAsPDF() {
     wikiHomePage.deleteWiki(title);
   }
 
+  /**
+   * <li>Case ID:122811.</li>
+   * <li>Test Case Name: Watch a wiki page of intranet.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> check issue WIKI-976 (wrong link in the mail)
+   */
+  /*
```

Comment:
No need to have 2 blocks of comments (this remark is valid for all comments blocks of this PR).

Reply:
ok